runtime/lib/descriptor/activity-descriptor.js

```diff
@@ -656,6 +656,52 @@ Object.assign(ActivityDescriptor.prototype,
       fn: function getMicMute () {
         return Promise.resolve(this._runtime.component.turen.muted)
       }
+    },
+    /**
+     * begin tts mix with audio.
+     *
+     * @memberof yodaRT.activity.Activity
+     * @instance
+     * @function mixTtsBegin
+     * @param {interrupt} [interrupt] interrupt audio if true, otherwise suppress audio. Undefined will use system config.
+     * @param {playerId} [number] playerId is required if the value of interrupt is false.
+     * @returns {Promise} Promise of audio mixed
+     */
+    mixTtsBegin: {
```

Comment:
I'd be curious what is the reason of this function presenting in activity API 🤔

Reply:
It needs to be used when tts and audio are played in parallel. Other applications and systems may also need to be used.

Reply:
Non activity related API could be implemented in packages under `@yodaos` scope.

Reply:
+1, What does the audio mix mean?

Reply:
This requires appId that can't let the app fill it out by itself.

Reply:
If the apis doesn't expected to be interacting with other apps, app id could be derived from caller automatically.

Reply:
Already move to package mediaKit.